docs/reviewer-guide.md

```diff
@@ -121,4 +121,5 @@ The lowest bar for acceptable golang **coding standards** (anti-patterns, coding
 
 The lowest **testing bar** to pass:
 * New code requires new unit tests.
-* New features and bug fixes require at least on e2e test (the core case must be tested).
+* New features require at least one e2e test. As they are costly: As many as needed, but as few as possible.
```

Comment:
This is vague to me. What one test do I need to write? Any, or empty? The previous wording described the reality pretty well. If you going to contribute a new feature you are required to provide tests covering  "core"/basic functionality. Further interesting edge cases can be tested/asked to be tested. We can still emphasize unit tests here, tho.

Reply:
To me the original wording was ... not helpful.

However, I'm restoring the old wording to unblock this pr.

Reply:
Would you mind elaborating on why you think so? It might be obvious that a new feature needs coverage by e2e tests but I witnessed the opposite.